src/main/java/com/dev/cinema/Main.java

```diff
@@ -64,38 +51,46 @@ public static void main(String[] args) throws AuthenticationException {
         fastSession.setShowTime(LocalDateTime.now().plusMonths(2));
         movieSessionService.add(fastSession);
 
-        movieSessionService.findAvailableSessions(gift.getId(), LocalDate.now())
-                .forEach(System.out::println);
         movieSessionService.findAvailableSessions(groundhogDay.getId(),LocalDate.now())
                 .forEach(System.out::println);
-        movieSessionService.findAvailableSessions(bobroPozhalovat.getId(),LocalDate.now())
+        movieSessionService.findAvailableSessions(fastAndFurious.getId(),LocalDate.now())
                 .forEach(System.out::println);
 
         movieService.getAll().forEach(System.out::println);
 
-        UserService userService = (UserService) injector.getInstance(UserService.class);
-        User den = new User();
-        den.setEmail("den@somemail.com");
-        den.setPassword("dent");
-        userService.add(den);
-
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         User johny = new User();
         johny.setEmail("johnydepp@amail.com");
         johny.setPassword("CaptainJackSparrow");
         authenticationService.register(johny.getEmail(), johny.getPassword());
-        authenticationService.login(johny.getEmail(), johny.getPassword());
+        johny = authenticationService.login(johny.getEmail(), johny.getPassword());
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        System.out.println(userService.findByEmail(johny.getEmail()).get());
+
+        User den = new User();
+        den.setEmail("den@somemail.com");
+        den.setPassword("dent");
+        authenticationService.register(den.getEmail(), den.getPassword());
+        den = authenticationService.login(den.getEmail(), den.getPassword());
         System.out.println(userService.findByEmail(johny.getEmail()).get());
 
         ShoppingCartService cartService = (ShoppingCartService) injector
                 .getInstance(ShoppingCartService.class);
+        System.out.println(cartService.getByUser(johny));
+
         cartService.addSession(fastSession, den);
         cartService.addSession(groundHogDayManSession, johny);
-        System.out.println("Den's shopping cart :" + cartService.getByUser(den).toString());
+
         System.out.println("Johny's shopping cart :" + cartService.getByUser(johny).toString());
+        System.out.println("Den's shopping cart :" + cartService.getByUser(den).toString());
```

Comment:
```suggestion
        System.out.println("Johny's shopping cart :" + cartService.getByUser(johny));
        System.out.println("Den's shopping cart :" + cartService.getByUser(den));
```
calling `toString()` explicitly is redundant